operator/cmd/mesh/testdata/manifest-generate/output/ztunnel.golden.yaml

```diff
@@ -220,6 +220,7 @@ spec:
             add:
             - NET_ADMIN
           privileged: true
+      priorityClassName: system-node-critical
```

Comment:
I think this needs to automatically be updated with a `make gen` command.

Reply:
`make gen` undid that change, so I removed it and squashed the commit.